cpp/src/node_options.cpp

```diff
@@ -162,6 +162,7 @@ class target_options::impl {
     option<enum target::expiry_policy> expiry_policy;
     option<std::vector<symbol> > capabilities;
     option<target::dynamic_property_map> dynamic_properties;
+    option<int> type;
```

Comment:
possibly replace this with `option<bool> is_coordinator;`?

Reply:
Done